pkgs/data/themes/levuaska-gtk-theme/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, gtk_engines }:
+
+stdenv.mkDerivation {
+  version = "2021-1-31";
+  pname = "levuaska-gtk-theme";
+
+  src = fetchFromGitHub {
+    owner = "selene57";
+    repo = "levuaska-gtk-theme";
```

```suggestion
    repo = pname;
```
This is pretty common across Nixpkgs. You have to change
```nix
stdenv.mkDerivation {
```
to
```nix
stdenv.mkDerivation rec {
```
for this to work.

---

I have amended the commit to fix this, thank you!